Packs/Hackuity/Integrations/Hackuity/Hackuity.py

```diff
@@ -0,0 +1,694 @@
+                "cvss": ["Score", "CVSS"],
+                "description": ["Description"],
+                "id": ["ID"],
+                "seen_first": ["Seen", "First"],
+                "trs": ["Score", "TRS"],
+            },
+            {
+                "seen_first": format_date,
+            },
+        )
+        output["Findings"] = json_format(
+            aggfinding["by_state"],
+            {
+                "total": ["Total"],
+                "open": ["Open"],
+                "closed": ["Closed"],
+                "ignored": ["Ignored"],
+            },
+        )
+        output["VulnTypes"] = [
+            {
+                "ID": vuln_type,
+                "Name": client.get_vulnerability_name(vuln_type),
+            }
+            for vuln_type in aggfinding["vuln_types"]
+        ]
+        outputs.append(output)
+    return CommandResults(
+        outputs_prefix="Hackuity.Vulnerabilities",
+        outputs=outputs,
+        readable_output=tableToMarkdown(
+            f"{'VulnDB' if hy_global_only else 'Provider'} vulnerabilities",
+            outputs,
+            headers=[
+                "VulnTypes",
+                "Description",
+                "Attributes",
+                "Score",
+                "Findings",
+                "Seen",
+            ],
+        ),
+    )
+
+
+def hackuity_dashboard_widgets_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    widgets = client.fetch_dashboard_widgets()
+    outputs = [
+        json_format(
+            widget,
+            {
+                "id": ["ID"],
+                "params": ["Params"],
+                "type": ["Type"],
+            },
+        )
+        for widget in widgets
+    ]
+    return CommandResults(
+        outputs_prefix="Hackuity.Dashboard.Widgets",
+        outputs=outputs,
+        readable_output=tableToMarkdown(
+            "Dashboard widgets",
+            outputs,
+            headers=["ID", "Type", "Params"],
+        ),
+    )
+
+
+def hackuity_dashboard_data_command(
+    client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    widget_id = args["widget_id"]
+    outputs = client.fetch_dashboard_widget_data(widget_id=widget_id)
+    return CommandResults(
+        outputs_prefix=f"Hackuity.Dashboard.Data.{widget_id}",
+        outputs=outputs,
+        readable_output=tableToMarkdown(
+            f"Dashboard widget data ({widget_id})",
+            outputs,
+        ),
+    )
+
+
+""" MAIN FUNCTION """
+
+
+def main() -> None:
+    """main function, parses params and runs command functions"""
+
+    url = demisto.params()["url"]
+    namespace = demisto.params()["namespace"]
+    login = demisto.params()["login"]["identifier"]
+    password = demisto.params()["login"]["password"]
+
+    verify_certificate = not demisto.params().get("insecure", False)
+    proxy = demisto.params().get("proxy", False)
+
+    commands: Dict[str, Callable[[Client, Dict[str, str]], CommandResults]] = {
```

Comment:
```suggestion
    commands = {
```

Reply:
I have added that type otherwise MyPy thinks it is the following:
```
builtins.dict[builtins.str*, builtins.object*]
```

…and fails on a later line with this error:

```
/Packs/Hackuity/Integrations/Hackuity/Hackuity.py:673:28: error:
"object" not callable  [operator]
                return_results(commands[command](client, demisto.args()))
```

Please feel free to suggest a better solution for MyPy!

Reply:
its okay you can keep it then